e2etests/Microsoft.Azure.Webjobs.Extensions.SignalRService.E2ETests/SimpleChatTests.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Client;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Xunit;
+using static Microsoft.Azure.Webjobs.Extensions.SignalRService.E2ETests.Utils;
+
+namespace Microsoft.Azure.Webjobs.Extensions.SignalRService.E2ETests
+{
+    public class SimpleChatTests
+    {
+        private const string Section = "FunctionBaseUrl:SimpleChat";
+        public static readonly SimpleChatClient Client = new();
+
+        public class BaseUrls : IEnumerable<object[]>
+        {
+            public static readonly IEnumerable<object[]> Data = from section in Configuration.GetSection(Section).GetChildren()
+                                                                select new object[] { section.Key, section.Value };
+
+            public IEnumerator<object[]> GetEnumerator() => Data.GetEnumerator();
+
+            IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+        }
+
+        public class NegotiationTests
+        {
+            [Theory]
+            [ClassData(typeof(BaseUrls))]
```

Comment:
Move e2e tests to `test` folder? And mark as `skip` or add some conditional attribute to skip normal gated UnitTests.